testing/lshell/APKBUILD

```diff
@@ -2,39 +2,35 @@
 # Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
 pkgname=lshell
 pkgver=0.9.18
-pkgrel=1
+pkgrel=2
 pkgdesc="Python-based limited Shell"
 url="http://ghantoos.org/limited-shell-lshell"
 arch="noarch"
 license="GPL3+"
 depends="python2"
-depends_dev=""
+options="!check"
```

Comment:
Does the package testsuite not work? Also, do the tests need to be packaged for a specific reason?